bundles/mapping/mapmodule/MapModuleButton.jsx

```diff
@@ -15,7 +15,6 @@ const Container = styled('div')`
 `;
 
 const StyledButton = styled(MapButton)`
-    opacity: 0.8;
```

Comment:
The current buttons are a bit transparent. So we should have this here. It has been moved to toolbar button instead. Should we have a MapButton with opacity if it doesn't have a toolbar and a toolbar one without opacity?